src/mark.js

```diff
@@ -280,7 +280,19 @@ class Mark { // eslint-disable-line no-unused-vars
         default:
             return `()(${str})`;
         case "complementary":
-            return `()([^\\s${lsJoin}]*${str}[^\\s${lsJoin}]*)`;
+            lsJoin = lsJoin.length ?
+                `^\\s${lsJoin}` :
+                [
+                    "A-Za-z\-", // Basic Latin + dash
+                    "\u00AD", // Soft hyphen
+                    "\u00AF", // Macron
+                    "\u00B4", // Acute accent
+                    "\u00B8", // Cedilla
+                    "\u00C0-\uD7FF",
+                    "\uF900-\uFDCF",
+                    "\uFDF0-\uFFEF"
```

Comment:
> Can you please also comment the last three entries?

I will, but they may not be very helpful because the unicode ranges cover a lot of characters, so I could keep the comment short and generalize, or try to add everything and make a comment that fills up 10 lines.

> Wouldn't it be helpful to double escape the unicode characters here? 

Ok, I'll fix that!

> Why are e.g. a macron or a cedilla characters that are necessary to capture?

Hmm, you're right... when I added them I was thinking they were combining characters, but they aren't. I'll remove them.